mem/mem.c

```diff
@@ -256,6 +256,10 @@ static struct {
           .mmap_thresh = 0,
           .nice = 0};
 
+#ifdef PER_THREAD_MALLOC
+static __thread comdb2ma *t_zone = NULL;
+#endif
+
```

Comment:
There's already a `#ifdef PER_THREAD_MALLOC` block just a few lines down here. Can this declaration be moved there?

Reply:
yes, thanks.